values.yaml

```diff
@@ -127,31 +120,23 @@ pip:
   initialDelaySeconds: 300 # pip service takes a long time to start up
   downloadCommand: "./bin/download --admin-only"
   limits:
-    memory: 10Gi
-    cpu: 3
-    # cannot use a hyphen to match the k8s param due to https://github.com/helm/helm/issues/2192
-    ephemeral_storage: 35Gi
+    memory: 12Gi
+    cpu: 2
   requests:
-    memory: 5Gi
-    cpu: 0.1
-    # cannot use a hyphen to match the k8s param due to https://github.com/helm/helm/issues/2192
-    ephemeral_storage: 35Gi
-  annotations: {}
-  nodeSelector: {}
-  pvc: {}
-#    create: true
-#    name: pip-pvc
-#    storageClass: aws-efs
-#    accessModes: ReadWriteMany
-#    storage: 10Gi
-#    annotations: {}
+    memory: 10Gi
+    cpu: 1
+  pvc:
+    create: true
+    name: pip 
+    storage: 45Gi
+    accessModes: ReadWriteOnce
+    storageClass: standard
 
 dashboard:
   enabled: true
   replicas: 1
   dockerTag: "latest"
-  domain: null # set this to enable an ingress
-  nodeSelector: {}
+  domain: dashboard.mydomain.com
```

Comment:
keep original. overriding values will be in another file that should not be committed to this repo